### DISCUSSION TITLE
Dexed as a TX81Z?

### DISCUSSION THREAD

**Discussion by beeltink** [asb2m10/dexed#461] [Q&A]
Are there any plans to add TX81Z functionality to Dexed?
A "DX7"-mode and a "TX81Z"-mode (4 operators, different wave-forms, 8 algorithms)?

**Comment by mtarenskeen** [comment]
The TX81Z (and family: DX100/27/21/11, WT11, YS100/200, TQ5, B200, DS55, V50) are more different from a DX7 than you might think. Let Dexed be restricted to emulate a DX7 as good as possible, nothing more, nothing less.

It is (sort of) possible to convert SysEx from 4-operator FM synths to DX7 SysEx for use in Dexed using [https://dxconvert.martintarenskeen.nl](https://dxconvert.martintarenskeen.nl). Or you can download ready to use converted 4-op patches from [https://4opdx7.martintarenskeen.nl](https://4opdx7.martintarenskeen.nl)
